CI script is failing because of new RO pipeline

Error in the GH Action:
```
make: *** No rule to make target 'other_import.owl'.  Stop.
```

Since the RO was updated to use ODK, the `other_imports.owl` was moved to be an import in the ODK way. It was moved to `imports/other_imports.owl`.

As part of the change of RO using ODK, `imports/other_imports.owl` only now have two external classes; all other content was moved to their specific imports.

Is adding this file to the pipeline still necessary?

--- a/ci/catalog-v001.xml
+++ b/ci/catalog-v001.xml
@@ -13,5 +13,4 @@
     <uri id="bspo-base" name="http://purl.obolibrary.org/obo/bspo/bspo-base.owl" uri="biological-spatial-ontology/src/ontology/bspo-base.owl"/>
     <uri id="obi-base" name="http://purl.obolibrary.org/obo/obi/obi-base.owl" uri="obi/views/obi-base.owl"/>
     <uri id="ro-base" name="http://purl.obolibrary.org/obo/ro/ro-base.owl" uri="obo-relations/src/ontology/ro-base.owl"/>
-    <uri id="ro-other-import" name="http://purl.obolibrary.org/obo/ro/other_import.owl" uri="obo-relations/src/ontology/other_import.owl"/>
 </catalog>

--- a/ci/test-merged.ofn
+++ b/ci/test-merged.ofn
@@ -15,7 +15,6 @@ Import(<http://purl.obolibrary.org/obo/uberon/uberon-base.owl>)
 Import(<http://purl.obolibrary.org/obo/cl/cl-base.owl>)
 Import(<http://purl.obolibrary.org/obo/bspo/bspo-base.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/ro-base.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/other_import.owl>)
 Import(<http://purl.obolibrary.org/obo/go/extensions/go-bfo-bridge.owl>)
 # This one has an odd IRI:
 Import(<http://purl.obolibrary.org/obo/uberon-bridge-to-bfo.owl>)

--- a/ci/test-merged.sh
+++ b/ci/test-merged.sh
@@ -42,7 +42,6 @@ cd $ROOT_DIR
 
 cd obo-relations/src/ontology
 make ro-base.owl
-make other_import.owl
 cd $ROOT_DIR
 
 cd obi

